packages/loader/driver-definitions/src/storage.ts

```diff
@@ -64,16 +64,6 @@ export interface IDocumentStorageService {
      */
     getVersions(versionId: string | null, count: number): Promise<IVersion[]>;
 
-    /**
```

Comment:
Here is what I'd suggest you do:
1. Keep read() signature here for now. For backward compatibility, all storage interfaces would need to have same read() implementation as it is today, and having it here will ensure compiler will check that we indeed continue to implement it correctly in all implementations.
2. Once everything is done, compiles and runs through UT and manual validation, remove it from here (but not from implementations) to force everyone taking new drop of Fluid Framework to move to readBlob() API. This can be done as separate (or same) PR and would need to have note in breaking.md.